### PR TITLE
[WIP] Add minio log storage for tekton-results.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Add minio log storage for tekton-results for development instances.

Demo: https://youtu.be/MHPFu7CnKvw

### CODE REVIEW

File: developer/ckcp/config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 cluster_type: openshift
 
 # git_url refers to a git repo to be considered as the source of truth for Argo CD applications.
-git_url: https://github.com/openshift-pipelines/pipeline-service.git
+git_url: https://github.com/AndrienkoAleksandr/pipelines-service.git
```
Comment (adambkaplan) [inline developer/ckcp/config.yaml:9]: Drop this once you are ready to remove it from the "draft" state.

File: developer/ckcp/openshift-operators/minio/tenant/tenant.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  features:
+    bucketDNS: false
+    domains: { }
+  # It doesn't work. TODO: create an issue for minio operator
+  # users:
+  #   - name: storage-user
+  # buckets:
+  #   - name: "tekton-results"
+  #     region: "us-east-1"
+  #     objectLock: true
```
Comment (adambkaplan) [inline developer/ckcp/openshift-operators/minio/tenant/tenant.yaml:26]: Can you describe what the issue is? Or do you plan to submit an issue upstream?

File: operator/gitops/argocd/argo-apps/cert-manager-operator.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ spec:
     server: https://kubernetes.default.svc
   source:
     path: operator/cert-manager
-    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
+    repoURL: https://github.com/AndrienkoAleksandr/pipelines-service.git
```
Comment (adambkaplan) [inline operator/gitops/argocd/argo-apps/cert-manager-operator.yaml:13]: Any reason why you need to test with your fork for cert-manager?

File: operator/gitops/argocd/argo-apps/tekton-chains.yaml [hunk 4]
```diff
@@ -11,7 +11,7 @@ spec:
     server: https://kubernetes.default.svc
   source:
     path: operator/gitops/argocd/tekton-chains/overlays/openshift
-    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
+    repoURL: https://github.com/AndrienkoAleksandr/pipelines-service.git
```
Comment (adambkaplan) [inline operator/gitops/argocd/argo-apps/tekton-chains.yaml:14]: Any reason why you need to test with your fork of pipelines?

**Review by adambkaplan — COMMENTED** [review]
An initial pass - good to showcase the overlays for tekton results.

File: operator/gitops/argocd/argo-apps/cert-manager-operator.yaml (hunk 3 above)
Comment (AndrienkoAleksandr) [reply operator/gitops/argocd/argo-apps/cert-manager-operator.yaml:13]: No, that's only for case if in the original repo was changed folder location due refactoring or someting like that.

File: developer/ckcp/openshift-operators/minio/tenant/tenant.yaml (hunk 2 above)
Comment (AndrienkoAleksandr) [reply developer/ckcp/openshift-operators/minio/tenant/tenant.yaml:26]: I submitted: https://github.com/minio/operator/issues/1345 and https://github.com/minio/operator/issues/1346

File: operator/gitops/argocd/argo-apps/tekton-chains.yaml (hunk 4 above)
Comment (AndrienkoAleksandr) [reply operator/gitops/argocd/argo-apps/tekton-chains.yaml:14]: No, that's only for case if in the original repo was changed folder location due refactoring or someting like that.

**Comment by AndrienkoAleksandr** [comment]
@adambkaplan pr is ready. But first should be merged https://github.com/openshift-pipelines/tektoncd-results/pull/18 and ideally https://github.com/openshift-pipelines/tektoncd-results/pull/19 too.   After that I need to use newer images for tekton results deployments.

**Comment by adambkaplan** [comment]
@AndrienkoAleksandr given our transition away from kcp, I think we should hold off on merging things new things into this repo until the infra team completes the refactor.

**Comment by AndrienkoAleksandr** [comment]
Closed on favour https://github.com/openshift-pipelines/pipeline-service/pull/419

**Pull request closed without merging**